src/views/monitoring/alert/modules/alert-list/AlertDataTable.vue

```diff
@@ -395,23 +406,23 @@ export default {
                     alert_id: state.selectedItems[0].alert_id,
                 });
                 await listAlerts();
-                showSuccessMessage('Alert Delete confirm', '', root);
+                showSuccessMessage(vm.$t('MONITORING.ALERT.ALERT_LIST.FORM.ALT_S_DELETE'), '', root);
             } catch (e) {
                 console.error(e);
-                showErrorMessage('Alert Delete failure', '', root);
+                showErrorMessage(vm.$t('MONITORING.ALERT.ALERT_LIST.FORM.ALT_S_DELETE'), e, vm.$root);
```

Comment:
showSuccessMessage 와 root 를 주는 방식을 통일하는게 좋을 것 같아요!
```suggestion
                showErrorMessage(vm.$t('MONITORING.ALERT.ALERT_LIST.FORM.ALT_S_DELETE'), e, root);
```